src/soc/ibm/power9/istep_10_10.c

```diff
@@ -161,14 +161,37 @@ static const size_t pec_lane_cfg_sizes[] = {
 };
 
 /*
- * PEC_PCIE_LANE_MASK_NON_BIFURCATED in processed talos.xml for the first
- * processor chip.  Values correspond to lane_width enumeration.
+ * Rows correspond to PEC_PCIE_LANE_MASK_NON_BIFURCATED values in processed
+ * talos.xml for each processor chip.  Values correspond to lane_width
+ * enumeration.
  */
-static uint16_t lane_masks[MAX_PEC_PER_PROC][MAX_LANE_GROUPS_PER_PEC] = {
+static uint16_t lane_masks_chip0[MAX_PEC_PER_PROC][MAX_LANE_GROUPS_PER_PEC] = {
 	{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
 	{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X8_GRP1,               0x0 },
 	{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X4_GRP0, LANE_MASK_X4_GRP1 },
 };
+static uint16_t lane_masks_chip1[MAX_PEC_PER_PROC][MAX_LANE_GROUPS_PER_PEC] = {
+	{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
+	{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X8_GRP1,               0x0 },
+	{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
+};
+
+static uint16_t (*lane_masks[])[MAX_LANE_GROUPS_PER_PEC]  = {
+	lane_masks_chip0,
+	lane_masks_chip1,
+};
```

Comment:
This is hard to read and gives unnecessary level of indirection.
```suggestion
static uint16_t lane_masks[][MAX_PEC_PER_PROC][MAX_LANE_GROUPS_PER_PEC]  = {
	{
		{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
		{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X8_GRP1,               0x0 },
		{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X4_GRP0, LANE_MASK_X4_GRP1 },
	},
	{
		{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
		{ LANE_MASK_X8_GRP0, 0x0, LANE_MASK_X8_GRP1,               0x0 },
		{ LANE_MASK_X16,     0x0,               0x0,               0x0 },
	},
};
```